src/integrationTest/java/com/weirddev/testme/intellij/generator/TestMeGeneratorTestBase.java

```diff
@@ -110,13 +109,14 @@ protected void doTest(final String packageName, String testSubjectClassName, fin
             myFixture.openFileInEditor(fooClass.getContainingFile().getVirtualFile());
 
             PsiElement result = new TestMeGenerator(new TestClassElementsLocator(), testTemplateContextBuilder,new CodeRefactorUtil()).generateTest(new FileTemplateContext(new FileTemplateDescriptor(templateFilename), language, getProject(),
-                    expectedTestClassName,
-                    targetPackage,
-                    getModule(),
-                    getModule(),
-                    srcDir,
-                    fooClass,
-                    fileTemplateConfig));
+                expectedTestClassName,
+                targetPackage,
+                getModule(),
+                getModule(),
+                srcDir,
+                fooClass,
+                fileTemplateConfig,
+                new FileTemplateCustomization(new ArrayList<>(), new ArrayList<>(), false)));
```

Comment:
regrading the previous comment about adding Integration test.... `new FileTemplateCustomization(...)` can be extracted as an argument of this doTest() method - that will be passed as default value by existing callers - currently 2 callers.
an overloaded version of doTest(FileTemplateCustomization) can be added that will pass default values for the arguments of this doTest(...) method.
please see overloaded `doTest(FileTemplateConfig)` as example for a similar use case